src/cpp/rtps/builtin/discovery/endpoint/EDPSimple.cpp

```diff
@@ -218,10 +212,6 @@ bool EDPSimple::createSEDPEndpoints()
         ratt.endpoint.multicastLocatorList = this->mp_PDP->getLocalParticipantProxyData()->m_metatrafficMulticastLocatorList;
         //ratt.endpoint.remoteLocatorList = m_discovery.initialPeersList;
         ratt.endpoint.durabilityKind = TRANSIENT_LOCAL;
-        ratt.times.heartbeatResponseDelay.seconds = 0;
-        ratt.times.heartbeatResponseDelay.fraction = 0;
-        ratt.times.initialAcknackDelay.seconds = 0;
-        ratt.times.initialAcknackDelay.fraction = 0;
         this->mp_pubListen = new EDPSimplePUBListener(this);
```

Comment:
Maybe use specific values for all the ratt.times fields? Even better, have a const for it at the top of the file?

Reply:
Done